.github/workflows/test.yml

```diff
@@ -191,12 +191,12 @@ jobs:
         name: Dump serve logs
         if: always() && matrix.skip-web-check != '1'
         run: |
-          cat /tmp/artisan-serve.log
+          [ ! -f /tmp/artisan-serve.log ] || cat /tmp/artisan-serve.log
```

Comment:
```suggestion
          [ -f /tmp/artisan-serve.log ] && cat /tmp/artisan-serve.log
```

Reply:
"if not else", is not as easy to follow :)

Reply:
`cat /tmp/artisan-serve.log || true`
is even dryer.

Reply:
I like the last one... I started with the second one, but it still returned 1 instead of 0 :D